templates/service-template.yml

```diff
@@ -285,6 +285,36 @@ parameters:
   description: A list of users that are allowed to create standard KAFKA instances via their organisation in a yaml format.
   value: "[{id: 13640203, max_allowed_instances: 5, any_user: true, registered_users: []}, {id: 12147054, max_allowed_instances: 1, any_user: true, registered_users: []}, {id: 13639843, max_allowed_instances: 1, any_user: true, registered_users: []}]"
 
+- name: TENANT_USERNAME_CLAIM
+  displayName: Token claims username key
+  description: Token claims key to retrieve the corresponding username.
+  value: "username" 
+
+- name: TENANT_ID_CLAIM
+  displayName: Token claims organisation ID key
+  description: Token claims key to retrieve the corresponding organisation ID.
+  value: "org_id" 
+
+- name: TENANT_ORG_ADMIN_CLAIM
+  displayName: Token claims organisation admin access key
+  description: Token claims key to retrieve the corresponding organisation admin role.
+  value: "is_org_admin" 
+
+- name: ALTERNATE_TENANT_USERNAME_CLAIM
+  displayName: Token claims alternate username key
+  description: Token claims key to retrieve the corresponding alternate username.
```

Comment:
Token claims key to retrieve the corresponding username (principal) using an alternative claim.